### PR TITLE
Feature crawler

### PR DESCRIPTION
**Pull request by flenkbm**
working crawler with not working starting file (source.py)

### CODE REVIEW

File: crawler/filecleaner.py
Comment (Minuta18) [inline crawler/filecleaner.py:1]: Create a function which named like a "create_temp()" or something like this. Your current code style is awful!

File: crawler/filemaker.py
Comment (Minuta18) [inline crawler/filemaker.py:1]: Why dont u merge this with crawler/filecleaner.py? Why there are no any docs? And please, move it at least in function

File: crawler/rawfile_editor.py
Comment (Minuta18) [inline crawler/rawfile_editor.py:1]: Split it all in functions. Current state of code is unacceptable

File: crawler/rawreader.py
Comment (Minuta18) [inline crawler/rawreader.py:1]: What the difference with rawreader.py and rawfile_editor.py?

File: crawler/sitegetter.py
Comment (Minuta18) [inline crawler/sitegetter.py:1]: Why does we need pywebcopy dependency?

File: crawler/source.py
Comment (Minuta18) [inline crawler/source.py:1]: It's awful. No comments.

**Review by Minuta18 — CHANGES_REQUESTED** [review]
Your code needs to be REWRITED. (And be up to PEP8). Please, use python's features like "classes" and "functions"!

**Comment by Minuta18** [comment]
Your code need to be finalized

**Comment by M-Stepan-373** [comment]
test

**Comment by M-Stepan-373** [comment]
Oh i can write here its good

**Pull request closed without merging**